runner/run/processors/port_processor.py

```diff
@@ -121,6 +125,37 @@ def _convert_to_path(val):
         file_obj['path'] = path
         return file_obj
 
+    @staticmethod
+    def _covert_to_cwl_format(val):
+        file_obj = copy.deepcopy(val)
+        location = file_obj.pop('location',None)
+        if location:
+            try:
+                file_db_object = FileProcessor.get_file_obj(location)
+            except FileHelperException as e:
+                raise PortProcessorException('File %s not found' % location)
+            path = file_db_object.path
+            if path:
```

Comment:
[File instance](https://github.com/mskcc/beagle/blob/master/file_system/models.py#L73) can't exist without path. null=False, and required=True is by default. So this condition will always be satisfied.